apps/studio/components/interfaces/Settings/Infrastructure/InfrastructureConfiguration/MapView.tsx

```diff
@@ -182,14 +181,16 @@ const MapView = ({
                 {selectedRegionKey === region.region && (
                   <circle
                     r={4}
-                    className={`animate-ping ${hasNoDatabases ? 'fill-white/30' : 'fill-brand'}`}
+                    className={`animate-ping ${
+                      hasNoDatabases ? 'fill-black/30 dark:fill-white/30' : 'fill-brand'
+                    }`}
                   />
                 )}
                 <circle
                   r={4}
                   className={`cursor-pointer ${
                     hasNoDatabases
-                      ? 'fill-background-surface-300 stroke-white/20'
+                      ? 'fill-background-surface-300 stroke-black/20 dark:stroke-white/20'
```

Comment:
you can use `stroke-border-**` btw

Reply:
i don't think its working it seems
![image](https://github.com/supabase/supabase/assets/19742402/922b5631-8b3f-47f5-ba33-5dd219e69fc8)

Reply:
ahh okay - i need to figure out what stroke-border-** are available
stroke-border-stronger works perhaps